api/tests/Feature/UserTest.php

```diff
@@ -2154,4 +2154,90 @@ public function testFilterByClassificationToSalaryWithPools(): void
             ]
         ]);
     }
+
+    public function testCountApplicantsQuery(): void
+    {
+        // Get the ID of the base admin user
+        $user = User::All()->first();
+
+        // Create new pools and attach to new pool candidates.
+        $pool1 = Pool::factory()->create([
+            'user_id' => $user['id']
+        ]);
+        $pool2 = Pool::factory()->create([
+            'user_id' => $user['id']
+        ]);
+
+        PoolCandidate::factory()->count(8)->create([
+            'pool_id' => $pool1['id'],
+            'expiry_date' => FAR_FUTURE_DATE,
```

Comment:
I know you didn't add this constant but could you move it inside the class to get rid of the warning when running tests?

Reply:
I wasn't getting any warning for this myself for some reason but I've moved it anyway.